documentation.md

```diff
@@ -0,0 +1,35 @@
+# Documentation Policies
+
+## Documentation
+
+The project documentation will reside on the `wiki` page or/and in the` doc` folder of the project repository.
+
+When a new feature is added or a significant change is made. Firstly, a design proposal must be created and only once it is accepted and implemented will be added as part of the project documentation.
+
+## Design Proposals
+
+The design proposal is to facilitate communication with team members and be clear about the idea about the implementation. 
```

Comment:
Change the following:
1. is to facilitate communication -> is to make communication easier
2. be clear about the idea about the implementation -> be clear about the idea and documentation.